history/characters/53000_hillsbrad.txt

```diff
@@ -446,4 +446,22 @@
 		}
 	}
 	620.3.13={ death=yes }
+}
+
+#Anetta Crank
+53100={
```

Comment:
She doesn't have a dynasty, is it ok?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81499552-7feeb700-92dd-11ea-8ad4-960bb5d25107.png)

</details>